### PR TITLE
Add event extra_keys description (fixes #585)

### LINKED ISSUE #585: Dictionary should show documentation of event extra_keys
The description at https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/events_browser_menu_action isn't as helpful as it could be because the extra_values described in the definition at https://github.com/mozilla-mobile/fenix/blob/7cdf7a54df15d96b6f68b73be8140abb2223109d/app/metrics.yaml#L193 hold most of the event's content.

The link to the definition is really great here, though!

### PR DESCRIPTION
**Pull request by Iinh**
Fixes #585.

Screenshot:

CleanShot 2021-05-17 at 08 38 50@2x

### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

**Review by wlach — CHANGES_REQUESTED** [review]
Looks great code-wise, I think putting it on top of the metrics page makes it a little overwhelming though.

Can you give this a full extra section (with a header) called "Extra Keys" (after "Definition") with a HelpHoverable next to it? The text should say:

"The acceptable keys on the "extra" object sent with events."

and the link should be the one you provided.

**Comment by Iinh** [comment]
@wlach Should we put the extra keys in a metadata table or just a separate section with a heading like below is good enough?

CleanShot 2021-05-17 at 10 41 06@2x

**Comment by wlach** [comment]
> @wlach Should we put the extra keys in a metadata table or just a separate section with a heading like below is good enough?

I was wondering about that. I think this is probably good enough to start, but if it's easy a table might be easier to skim.

File: src/pages/MetricDetail.svelte [hunk 1]
```diff
@@ -117,6 +119,25 @@
     {/each}
     ping{metric.send_in_pings.length > 1 ? 's' : ''}.
   </p>
+
+  {#if metric.extra_keys && !isEmpty(metric.extra_keys)}
+    <h2>
+      Extra key{Object.keys(metric.extra_keys).length > 1 ? 's' : ''}
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:125]: Looking at your screenshot, I think it makes more sense to just always use the plural. This really is a group of things, and the plural makes that more clear.

```suggestion
      Extra keys
```

**Review by wlach — CHANGES_REQUESTED** [review]
Looking good! Let's see what it looks like when we turn this into a table.

**Comment by Iinh** [comment]
With tables: https://deploy-preview-597--glean-dictionary-dev.netlify.app/apps/fenix/metrics/events_app_opened_all_startup


CleanShot 2021-05-17 at 11 40 10@2x

Regular:

CleanShot 2021-05-17 at 11 41 09@2x

@wlach wdyt?

File: src/pages/MetricDetail.svelte [hunk 2]
```diff
@@ -117,6 +119,28 @@
     {/each}
     ping{metric.send_in_pings.length > 1 ? 's' : ''}.
   </p>
+
+  {#if metric.extra_keys && !isEmpty(metric.extra_keys)}
+    <h2>
+      Extra keys
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:125]: Can you put this after "Definition" below?

**Review by wlach — APPROVED** [review]
I think the table looks good! Just one minor change request.

**Pull request merged**